webpack-subresource-integrity-embroider/index.js

```diff
@@ -33,6 +33,12 @@ class SubresourceIntegrityPlugin {
               element.tagName === "SCRIPT"
                 ? element.getAttribute("src")
                 : element.getAttribute("href");
+
+            if (!assetLocation) {
+              // skip inline scripts
+              return;
+            }
```

Comment:
I think this is mostly for catching inline scripts. Having link elements without href probably rarely happens.

But in both cases I think it's safe to ignore integrity hash calculation...